library/src/main/scala/g8.scala

```diff
@@ -462,35 +474,45 @@ object G8 {
     }
   }
 
-  def readProps(stm: java.io.InputStream): G8.OrderedProperties = {
-    val p = new LinkedListProperties
-    p.load(stm)
-    stm.close()
-    (OrderedProperties.empty /: p.keyList) { (l, k) =>
-      l :+ (k -> p.getProperty(k))
+  def readProps(stm: InputStream): G8.OrderedProperties = {
+    // Overrides java.util.Properties to return properties back in the order they were specified
+    val properties = new Properties {
+      val parameters = mutable.MutableList.empty[(String, String)]
+
+      override def put(key: Object, value: Object) = {
+        val k = key.toString
+        val v = value.toString
+
+        parameters += k -> v
+        super.put(k, v)
+      }
+    }
+
+    try {
+      properties.load(stm)
+      properties.parameters.toList
+    } finally {
+      stm.close()
     }
   }
+
+  private val truthy = Set("y", "yes", "true")
+  private val falsy = Set("n", "no", "false")
```

Comment:
The problem is that this is a potentially breaking change: if a user inputs `off` for a string field, it will be rendered as `false` in the template. I discussed the issue with @eed3si9n and we came to the conclusion that `yes`, `n`, etc. are unlikely to be used that way. But each alternative we add increases the chance of conflict. I think this is a very reasonable default set of options.